ts/components/wallet/WalletLayout.tsx

```diff
@@ -44,17 +47,17 @@ export enum CardEnum {
 
 type FullCard = Readonly<{
   type: CardEnum.FULL;
-  card: CreditCard;
+  card: Wallet;
```

Comment:
for consistency, let's call it `wallet` instead of `card` then :)

Reply:
In this very specific case, I think it would make more sense to keep the "card" name, since what is being referred to here is the actual "card" component (the gray rectangle w/ the payment method information). Pretty much everywhere else, I updated the references to "card" to "wallet".

I'm also creating a technical debt story to define when the naming should be "card"-based, and when "wallet"-based (e.g. when adding a new *card*, it should be called a 'card', but when showing a list of payment methods, it should be 'wallet').